esp/templates/errors/program/notastudent.html

```diff
@@ -16,7 +16,7 @@ <h1>{{prog.niceName }} Student Registration&mdash; Error</h1>
 Please note: <em>Each</em> child should create a separate account for her-/himself. This allows us to accurately determine which students are in which classes.
 <br />
 <br />
-<a href="/myesp/signout/?redirect=/myesp/register/&role=Student" title="Create a student account!">Click here to create a new account for an ESP Student!</a>
+<a href="/myesp/signout/?redirect=/myesp/register/&role=Student" title="Create a student account!">Click here to create a new {{ settings.INSTITUTION_NAME }} {{ settings.ORGANIZATION_SHORT_NAME }} student account!</a>
```

Comment:
Again, maybe just the short name?  "create a new Berkeley Splash student account" seems weirdly verbose.

Reply:
That makes sense